examples/transformerconfigs/README.md

```diff
@@ -606,3 +606,64 @@ metadata:
     foo.k8s.io/bar: baz
 ```
 Kustomize supports escaping special characters in path, e.g `metadata/annotations/foo.k8s.io\/bar`
+
+## Namespace
```

Comment:
nit: move this section to right below the AnnotationsTransformer, so that the simpler transformers are all together.